phenotrex/cli/predict.py

```diff
@@ -10,6 +8,13 @@
               required=False, help='Input genotype file.')
 @click.option('--classifier', required=True, type=click.Path(exists=True),
               help='Path of pickled classifier file.')
+@click.option('--out_explain_per_sample', type=click.Path(dir_okay=False),
+              help='Path to file to which to write SHAP explanations for each predicted samples. Optional.')
```

Comment:
I'd prefer more concise help messages, like `'Write SHAP explanations for each predicted sample to file (optional).'`.